lgsm/config-default/config-lgsm/gmodserver/_default.cfg

```diff
@@ -25,17 +25,15 @@ wscollectionid=""
 
 ## Custom Start Parameters
 # Default -disableluarefresh, disables lua autorefresh reducing server lag. Auto refresh only useful for developers.
-customparms="-disableluarefresh"
+customstartparameters="-disableluarefresh"
 
 ## Game Server Login Token (GSLT): Optional
 # GSLT can be used for running a public server.
 # More info: https://docs.linuxgsm.com/steamcmd/gslt
 gslt=""
 
 ## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
-fn_parms(){
-parms="-game garrysmod -strictportbind -ip ${ip} -port ${port} -tickrate ${tickrate} +host_workshop_collection ${wscollectionid} -authkey ${wsapikey} +clientport ${clientport} +tv_port ${sourcetvport} +gamemode ${gamemode} +map ${defaultmap} +sv_setsteamaccount ${gslt} +servercfgfile ${servercfg} -maxplayers ${maxplayers} ${customparms}"
-}
+startparameters="-game garrysmod -strictportbind -ip ${ip} -port ${port} -tickrate ${tickrate} +host_workshop_collection ${wscollectionid} -authkey ${wsapikey} +clientport ${clientport} +tv_port ${sourcetvport} +gamemode ${gamemode} +map ${defaultmap} +sv_setsteamaccount ${gslt} +servercfgfile ${servercfg} -maxplayers ${maxplayers} ${customparms}"
```

Comment:
Above changed variable name not in the startparameter

Reply:
gmod only server using cutom start parameters. Unsure why this exists